### ISSUE TITLE
Regression problem for user simulations

### ISSUE THREAD

**Issue by michal-stolarz** [TsiakasK/sequence-learning-dataset#1]
Dear Mr Tsiakas,
I am a master's student and I am very interested in the results obtained by you. I think I spotted a problem with the training of neural networks for the purpose of creating the performance user simulation. In the paper (https://www.mdpi.com/2227-7080/6/2/49) you mentioned that "Since the purpose of the user simulation is not to develop a generalized model for task performance
and engagement, but to represent our collected data as accurately as possible". After looking into the code, where the neural network is trained for 10000 epochs I understood that the aim is to overfit it. However, I saw that the activation function in the used architecture (sigmoid) and lack of z normalization of the input data might have prevented it and result in not so good regression (for example for the 3rd cluster https://github.com/TsiakasK/sequence-learning-dataset/blob/master/simulation/performance_c2.png). I would like to ask you a question if it was done on purpose and if the model that better fits the data (e.g. gaussian process) might be better for the design of the user simulation?
Kind regards
Michał Stolarz
